yt_dlp/extractor/patreon.py

```diff
@@ -147,60 +196,155 @@ def _real_extract(self, url):
                         'uploader_url': user_attributes.get('url'),
                     })
 
-        if not info.get('url'):
-            # handle Vimeo embeds
-            if try_get(attributes, lambda x: x['embed']['provider']) == 'Vimeo':
-                embed_html = try_get(attributes, lambda x: x['embed']['html'])
-                v_url = url_or_none(compat_urllib_parse_unquote(
-                    self._search_regex(r'(https(?:%3A%2F%2F|://)player\.vimeo\.com.+app_id(?:=|%3D)+\d+)', embed_html, 'vimeo url', fatal=False)))
-                if v_url:
-                    info.update({
-                        '_type': 'url_transparent',
-                        'url': VimeoIE._smuggle_referrer(v_url, 'https://patreon.com'),
-                        'ie_key': 'Vimeo',
-                    })
+        # handle Vimeo embeds
+        if try_get(attributes, lambda x: x['embed']['provider']) == 'Vimeo':
+            embed_html = try_get(attributes, lambda x: x['embed']['html'])
+            v_url = url_or_none(compat_urllib_parse_unquote(
+                self._search_regex(r'(https(?:%3A%2F%2F|://)player\.vimeo\.com.+app_id(?:=|%3D)+\d+)', embed_html, 'vimeo url', fatal=False)))
```

Comment:
Maybe vimeo's _extract_url should be used?

Reply:
potentially. don't know what embed html is though. would need to find a post with a vimeo video to test with.